pkg/collector/corechecks/oracle/oracle_t.go

```diff
@@ -0,0 +1,100 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package oracle
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	go_ora "github.com/sijms/go-ora/v2"
+
+	_ "github.com/godror/godror"
+)
+
+var chk Check
+
+var HOST = "localhost"
+var PORT = 1521
+var USER = "c##datadog"
+var PASSWORD = "datadog"
+var SERVICE_NAME = "XE"
+var TNS_ALIAS = "XE"
+var TNS_ADMIN = "/Users/nenad.noveljic/go/src/github.com/DataDog/datadog-agent/pkg/collector/corechecks/oracle/testutil/etc/netadmin"
+
+func TestBasic(t *testing.T) {
+	chk = Check{}
+
+	// language=yaml
+	rawInstanceConfig := []byte(fmt.Sprintf(`
+server: %s
+port: %d
+username: %s
+password: %s
+service_name: %s
+tns_alias: %s
+tns_admin: %s
+`, HOST, PORT, USER, PASSWORD, SERVICE_NAME, TNS_ALIAS, TNS_ADMIN))
+
+	err := chk.Configure(integration.FakeConfigHash, rawInstanceConfig, []byte(``), "oracle_test")
+	require.NoError(t, err)
+
+	assert.Equal(t, chk.config.InstanceConfig.Server, HOST)
```

Comment:
Instead of a series of `assert.Equal`, this could/should just be a single `assert.Equal(t, chk.config.InstanceConfig, config.InstanceConfig{Port: ...})`.